app/models/concerns/thredded/post_common.rb

```diff
@@ -30,6 +30,10 @@ module PostCommon
           SELECT MAX(p2.created_at) from #{posts_table_name} p2 WHERE p2.postable_id = #{posts_table_name}.postable_id)
         SQL
         )
+
+        # When we can't load a preloader, return the results as-is
+        return result if preloader.empty?
```

Comment:
I'm not yet sure what the preloaders are for or why we can no longer find them when a post has been reported. This is a hack that needs addressed.